Modules/Microsoft365DSC/DSCResources/MSFT_AADAuthenticationMethodPolicy/MSFT_AADAuthenticationMethodPolicy.schema.mof

```diff
@@ -24,6 +24,12 @@ class MSFT_AADAuthenticationMethodPolicyIncludeTarget
     [Write, Description("The kind of entity targeted. Possible values are: user, group."), ValueMap{"user","group","unknownFutureValue"}, Values{"user","group","unknownFutureValue"}] String TargetType;
 };
 [ClassVersion("1.0.0")]
+class MSFT_MicrosoftGraphExcludeTarget
```

Comment:
I believe the class is already in there as MSFT_AADAuthenticationMethodPolicyExcludeTarget, but the reference is wrong.

Reply:
Is not really the same. At least the name is different.

Reply:
The MSFT_MicrosoftGraphExcludeTarget is used by AuthenticationMethodsRegistrationCampaign and was missing. The other MSFT_AADAuthenticationMethodPolicyExcludeTarget was auto created by the tool.